### DISCUSSION TITLE
how open projact Wasm to android

### DISCUSSION THREAD

**Discussion by AbdalaMask** [unoplatform/uno#7603] [Q&A]
### Current behavior

how open projact Wasm to android

### Expected behavior

how open projact Wasm to android

### Works on UWP/WinUI

Yes

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia

### NuGet package version(s)

4.0.8

### Affected platforms

WebAssembly

### IDE

Visual Studio 2022

### IDE version

17

**Comment by jeromelaban** [comment]
Thank for your question. We need more details about what you're trying to do to answer it.

**Comment by AbdalaMask** [comment]
open browser Wasm to android

**Comment by AbdalaMask** [comment]
https://marketplace.visualstudio.com/items?itemName=vs-publisher-1448185.ConveyorbyKeyoti

I use it to generate an IP to unlock on the phone
How do I open and browse the project without it on the phone

**Reply by AbdalaMask** [reply]
Conveyor by Keyoti

**Reply by jeromelaban** [reply]
We're not providing tools to debug applications on Android, Uno only provides tooling support for desktop browsers.
